src/zoning-district-class/zoning-district-class.service.spec.ts

```diff
@@ -2,10 +2,16 @@ import { Test } from "@nestjs/testing";
 import { ZoningDistrictClassRepositoryMock } from "test/zoning-district-class/zoning-district-class.repository.mock";
 import { ZoningDistrictClassService } from "./zoning-district-class.service";
 import { ZoningDistrictClassRepository } from "./zoning-district-class.repository";
-import { getAllZoningDistrictClassesQueryResponseSchema } from "src/gen";
+import {
+  getAllZoningDistrictClassesQueryResponseSchema,
+  getZoningDistrictClassesByIdQueryResponseSchema,
+} from "src/gen";
+import { ResourceNotFoundException } from "src/exception";
 
 describe("zoning district class service unit", () => {
   let zoningDistrictClassService: ZoningDistrictClassService;
+  const zoningDistrictClassRepositoryMock =
+    new ZoningDistrictClassRepositoryMock();
 
   beforeEach(async () => {
     const zoningDistrictClassRepositoryMock =
```

Comment:
(continued from comment above) ...this existing definition is incorrectly scoped and is now redundant. Please provide a community service by removing this incorrect instance.